src/components/data-management/ProjectCard.jsx

```diff
@@ -58,7 +61,11 @@ const ProjectCard = (props) => {
 
   const updateProjectName = (newName) => {
     dispatch(updateProject(project.uuid, { name: newName.trim() }));
-    if (projects.meta.error === false) {
+
+    if (
+      (config.currentApiVersion === api.V1 && projects.meta.error === false)
+      || config.currentApiVersion === api.V2
+    ) {
       dispatch(updateExperiment(projectExperiment, { name: newName.trim() }));
     }
```

Comment:
Small fix: trigger `updateExperiment` if we are using api v2 (to send the api request, updateProject doesn't trigger any v2 requests)

Reply:
Sure, but then why are you triggering `updateExperiment` for api.V1 too?

Reply:
Becuase api v1 doesn't use updateExperiment, it sends a request over `updateProject`